### ISSUE TITLE
Relative `font-size` units for resizable text

### ISSUE THREAD

**Issue by Malvoz** [Leaflet/Leaflet#7548]
## Motivation

Text should be resizable to meet the requirements of <span title="Web Content Accessibility Guidelines">WCAG</span> 2.1 <span title="Success Criterion">SC</span> [1.4.4 Resize text](https://www.w3.org/TR/WCAG21/#resize-text).

This would require changing to _relative_ `font-size` units for _text_ content.

## Proposed solution

Change the following to use relative `font-size` units (e.g. `rem`, `em`, `%`, `smaller`, `larger`):

https://github.com/Leaflet/Leaflet/blob/0f904a515879fcd08f69b7f51799ee7f18f23fd8/dist/leaflet.css#L270

https://github.com/Leaflet/Leaflet/blob/0f904a515879fcd08f69b7f51799ee7f18f23fd8/dist/leaflet.css#L421

https://github.com/Leaflet/Leaflet/blob/0f904a515879fcd08f69b7f51799ee7f18f23fd8/dist/leaflet.css#L434

## Additional context

PR: https://github.com/Leaflet/Leaflet/pull/7571

**Comment by Chandu-4444** [comment]
Hey, I'm a new one here... I think I can work on this issue. Would you allow me to do this?

**Comment by Malvoz** [comment]
@Chandu-4444 Fine with me! Sorry for the delayed response.

**Comment by Falke-Design** [comment]
Closed by #7800
